tests/test_pylettize.py

```diff
@@ -0,0 +1,35 @@
+"""Pylettize, main test module."""
+
+import operator
+
+import numpy as np
+
+from pylettize.pylettize import dist_to_color, hex_to_rgb
+
+
+def test_converting_hex_to_rgb():
+    """Ensure a given hex rgb string is converted to an RGB f32 vector."""
+    to_test = {
+        "#ff0000": np.array([1, 0, 0], dtype=np.float32),
+        "#00ff00": np.array([0, 1, 0], dtype=np.float32),
+        "#0000ff": np.array([0, 0, 1], dtype=np.float32),
+        "#ffff00": np.array([1, 1, 0], dtype=np.float32),
+        "#00ffff": np.array([0, 1, 1], dtype=np.float32),
+        "#ff00ff": np.array([1, 0, 1], dtype=np.float32),
+    }
+
+    for hex, expected_value in to_test.items():
+        np.testing.assert_almost_equal(hex_to_rgb(hex), expected_value)
```

Comment:
This could certainly be parametrized at some point. Maybe this could be handled in some future test-oriented PR.

Reply:
See #10

Reply:
You could test it more thoroughly by getting predefined color hexes and RGB values of mixed colors. But the edge cases might be sufficient in this case.